pages/_app.js

```diff
@@ -1,16 +1,91 @@
 import App from 'next/app';
 import Head from 'next/head';
-import React from 'react';
+import React, { useEffect } from 'react';
 import { GlobalStyle } from '@catho/quantum';
+import Image from 'next/image'
+import logoImage from '../static/imgs/cathoLogo.png'
+import backgroundImg from '../static/imgs/background-image.png'
+
+
 
 export default class HomePage extends App {
+
+  componentDidMount() {
+
```

Comment:
why all this spaces? How about of keep the code closer? 
Just other codes here..